website/docs/docs/cloud/about-cloud-develop-defer.md

```diff
@@ -9,16 +9,13 @@ pagination_next: "docs/cloud/cloud-cli-installation"
 
 [Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). This is powered by using a production manifest for comparison, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts. 
```

Comment:
I know this wasn't part of your change but I couldn't help myself. Making sentence active.
```suggestion
[Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). dbt powers this by using a production manifest for comparison, and resolves the `{{ ref() }}` function with upstream production artifacts. 
```

Reply:
brilliant, thank you!